src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php

```diff
@@ -59,7 +59,7 @@ protected function configure()
             ])
             ->setDescription('Clear the cache')
             ->setHelp(<<<'EOF'
-The <info>%command.name%</info> command clears the application cache for a given environment
+The <info>%command.name%</info> command clears and warmup the application cache for a given environment
```

Comment:
lang: and warms up

Reply:
updated

Reply:
Shall we change the class doc too ? Currently we have `Clear and Warmup the cache.`